nfsstat/tests/common.py

```diff
@@ -0,0 +1,41 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+
+from datadog_checks.dev import get_here
+
+COMPOSE_FILE = os.path.join(get_here(), 'compose', 'compose.yaml')
+
+CONFIG = {
+    "init_config": {"nfsiostat_path": "docker exec nfs-client /usr/sbin/nfsiostat"},
+    "instances": [{"tags": ["tag1:value1"]}],
+}
+
+E2E_METADATA = {
+    'start_commands': [
+        'apt-get update',
+        'apt-get install -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" -y docker.io',
+    ],
+    'docker_volumes': ['/var/run/docker.sock:/var/run/docker.sock'],
```

Comment:
Why do we need to mount `docker` socket and install `docker.io` ?

Reply:
Aren't we supposed to mount the nfs share inside the docker container and use the embedded `/opt/datadog-agent/embedded/bin//nfsiostat-sysstat` to get stats about the mounted NFS ?

Reply:
We need to call the `nfsiostat` binary in the `nfs-client` container from the agent container, so we need to do `docker exec nfs-client nfsiostat` inside the agent container. That's why we need this. I used what is done for `cassandra_nodetool` https://github.com/DataDog/integrations-core/blob/master/cassandra_nodetool/tests/conftest.py#L17

Reply:
Can't mount an nfs filesystem in the agent container, see https://github.com/DataDog/integrations-core/pull/6652#discussion_r426473271